keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/KeelRepository.kt

```diff
@@ -127,7 +128,7 @@ interface KeelRepository {
 
   fun getResourcesByApplication(application: String): List<Resource<*>>
 
-  fun getSummaryByApplication(application: String): List<ResourceSummary>
```

Comment:
Are we not supporting this anymore?

Reply:
Not for resource summaries. As part of working on the API changes to support the new UI, we agreed that we should start enforcing (kinda) the assumption that we only support one delivery config per application, so you'll notice that all the methods called by `ApplicationController` to return the environment, artifact and resource summaries pass in a `DeliveryConfig`, not an application name.

Having said that, this assumption is now easier to remove if we decide to: we can just list the delivery configs by application name, and then include the results for each delivery config in the response, whereas before they would all probably have just been lumped together without any logical grouping.

Reply:
I see, thanks for explaining!